charts/prometheus/templates/prometheus-config-configmap.yaml

```diff
@@ -374,11 +375,72 @@ data:
         relabel_configs:
           - action: labelmap
             regex: __meta_kubernetes_service_label_(.+)
-          {{- if .Values.global.airflowOperator.enabled }}
           - source_labels: [__meta_kubernetes_service_label_astronomer_io_platform_release]
-            regex: ^\s*{{ .Release.Name }}\s*$
+            regex: ^{{ .Release.Name }}$
+          - source_labels: [__meta_kubernetes_service_annotation_prometheus_io_scrape]
             action: keep
+            regex: true
+          - source_labels: [__address__, __meta_kubernetes_service_annotation_prometheus_io_port]
+            action: replace
+            regex: ([^:]+)(?::\d+)?;(\d+)
+            replacement: $1:$2
+            target_label: __address__
+        metric_relabel_configs:
+          - source_labels: [instance]
+            regex: "([-a-z0-9]*[a-z0-9])-.*?"
+            replacement: "$1"
+            target_label: deployment
+            # Drop metrics that are task or dag specific
+          - source_labels: [__name__]
+            regex: 'airflow_task_instance_created_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_task_removed_from_dag_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_task_restored_to_dag_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_duration_success_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_duration_failed_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_dependency_check_.*_count'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_dependency_check_.*_sum'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_schedule_delay_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_duration_success_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dagrun_duration_failed_.*'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dag_processing_last_duration_.*_count'
+            action: drop
+          - source_labels: [__name__]
+            regex: 'airflow_dag_processing_last_duration_.*_sum'
+            action: drop
```

Comment:
we discussed to move this to helper templates and add for both airflow and operator scrapes